pkgs/applications/audio/klystrack/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub
+, SDL2, SDL2_image
+, pkgconfig
+}:
+
+stdenv.mkDerivation rec {
+  pname = "klystrack";
+  version = "1.7.6";
+
+  src = fetchFromGitHub {
+    owner = "kometbomb";
+    repo = "klystrack";
+    rev = "${version}";
+    fetchSubmodules = true;
+    sha256 = "1h99sm2ddaq483hhk2s3z4bjbgn0d2h7qna7l7qq98wvhqix8iyz";
+  };
+
+  buildInputs = [
+    SDL2.dev SDL2_image
+  ];
+  nativeBuildInputs = [ pkgconfig ];
+
+  buildPhase = ''
+    make PREFIX=$out CFG=release
```

Comment:
Suggestion:
`makeFlags = [ "PREFIX=${placeholder "out"}" "CFG=release" ];`
This enables to use preBuild and postBuild in overrides.